doc/cabal-project.rst

```diff
@@ -127,8 +127,10 @@ project are:
 .. cfg-field:: extra-packages: package list with version bounds (comma separated)
     :synopsis: Adds external packages as local
 
-    :strike:`Specifies a list of external packages from Hackage which
-    should be considered local packages.` (Not implemented)
+    Specifies a list of external packages from Hackage, which
+    should be considered local packages. The motivation for
+    :cfg-field:`extra-packages` is making libraries that are not
```

Comment:
Is this one possible motivation or *the* motivation?

Reply:
Taken straight from the horse's mouth:
https://github.com/haskell/cabal/issues/3737#issuecomment-243718434

I imagine there are other uses, but I don't know. 

"A common use case" would be an alternative.

Reply:
Just found https://github.com/haskell/cabal/issues/6952 and the related https://github.com/haskell/cabal/issues/7865 and it seems you can install extra executables with ``extra-packages`` too, but there are caveats. Not sure if it should go into the documentation, while those are unresolved.